family_context/settings.py

```diff
@@ -38,6 +39,48 @@
     default="https://127.0.0.1",
 )
 
+LOG_LEVEL = config("LOG_LEVEL", default="INFO")
+LOG_DESTINATION = config("LOG_DESTINATION", default="file_output")
+choose_log_handler = {"file_output": "file", "console_output": "console"}
+
+# By default, log to console
+LOGGING = {
+    "version": 1,
+    "disable_existing_loggers":False,
+    "handlers": {
+        "console": {
+           'level': LOG_LEVEL ,
+           'class': 'logging.StreamHandler',
+           'stream': sys.stdout,
+           'formatter': 'verbose'
+       },
+    },
+    "loggers": {
+        "core": {
+            "handlers": ["console"],
+            "level": LOG_LEVEL,
+            "propagate": True,
+        },
+    },
+    "formatters": {
+        "verbose": {
+            "format": "%(asctime)s [%(levelname)s] - %(message)s",
+            "datefmt": "%Y-%m-%d %H:%M:%S",
+        },
+    },
+}
+
+# add other logging output options.
+if LOG_DESTINATION:
```

Comment:
Since the default value of `LOG_DESTINATION` is `file_output` this condition will always resolve to true - so no need for the if statement

Reply:
The default log destination should be console, will comment above.

Reply:
still, the if statement will have no purpose in this context.

Reply:
Fixed now.